README.md

```diff
@@ -1,5 +1,45 @@
+
+Then [create a new project](http://apostrophecms.org/docs/tutorials/getting-started/creating-your-first-project.html) on your computer and work on building your beautiful website.
+
+The ApostropheCMS documentation will walk you through [editing page templates](http://apostrophecms.org/docs/tutorials/getting-started/editing-page-templates.html), and [global content](http://apostrophecms.org/docs/tutorials/getting-started/global.html), [creating custom widgets](http://apostrophecms.org/docs/tutorials/getting-started/custom-widgets.html) and [navigation](http://apostrophecms.org/docs/tutorials/getting-started/building-navigation.html), along with [configuring reusable "pieces"](http://apostrophecms.org/docs/tutorials/getting-started/reusable-content-with-pieces.html) and [global settings](http://apostrophecms.org/docs/tutorials/getting-started/settings.html). Plus, a whole lot more!
+
+## Taking it Further
+
+Our documentation also covers advanced topics such as, [managing permissions](http://apostrophecms.org/docs/tutorials/intermediate/permissions.html), [working with ApostropheCMS data](http://apostrophecms.org/docs/tutorials/intermediate/model-layer.html), [building forms](http://apostrophecms.org/docs/tutorials/intermediate/forms.html), and [hosting an ApostropheCMS site in production](http://apostrophecms.org/docs/tutorials/intermediate/deployment.html).
 
-# Apostrophe
+Moreover, there's tons of great info in the ["How To"](http://apostrophecms.org/docs/tutorials/howtos/index.html) section of docs, and it's always growing!
 
-Apostrophe is a CMS framework for nodejs that supports in-context editing, schema-driven content types, flexible widgets and a great deal more. This module contains everything necessary to build a website with Apostrophe. Please see the [Apostrophe documentation](http://apostrophenow.org) for complete information about building sites with Apostrophe.
+## Extending the Core with Modules
+
+ApostropheCMS is Node.js all the way through, which means it can be [extended](http://apostrophecms.org/extend) in all sorts of interesting ways. Here's a list of some of our favorite modules:
+
+* [apostrophe-blog](https://github.com/punkave/apostrophe-blog) - Everything you need to put a blog on your site.
+* [apostrophe-events](https://github.com/punkave/apostrophe-events) - Manage upcoming events.
+* [apostrophe-workflow](https://github.com/punkave/apostrophe-workflow) - Add powerful localization capabilities to your project.
+* [apostrophe-passport](https://github.com/punkave/apostrophe-passport) - Authenticate via Twitter, Facebook, Github and more.
+* [apostrophe-places](https://github.com/punkave/apostrophe-places) - Manage geotagged content and display it on a map.
+* [apostrophe-redirects](https://github.com/punkave/apostrophe-redirects) - An easy redirects module.
+
+## Additional Resources
+We've got some great open-source projects that can help you out deploying, managing and maintaining Apsotrophe sites.
```

Comment:
"Apsotrophe" -> typo. Also, just point to the current "apostrophe in production" article on the site, these recommendations could get stale.